app/assets/stylesheets/sufia/_form-progress.scss

```diff
@@ -47,3 +47,9 @@ aside.form-progress {
     }
   }
 }
+
```

Comment:
Moved the custom CSS from fixedsticky to here and added another class to target the save widget and not all fixedsticky elements